README.md

```diff
@@ -34,9 +34,15 @@ A bit more about the different fields...
 - `allow_private` (optional)
     - bool [true, false]
     - default: false
-    - If false bugs that are not public will not be synchronized. Note that in order to synchronize
+    - If `false` bugs that are not public will not be synchronized. Note that in order to synchronize
       private bugs the bugzilla user that JBI runs as must be in the security groups that are making
       the bug private.
+- `brackets_required` (optional)
+    - bool [true, false]
+    - default: true
+    - If `true` the whiteboard_tag must be be contained within an opening ([) and closing (]) bracket, this is the default behavior; i.e. this tag `XYX` matches this whiteboard_tag `[XYZ]` but not `XYZ`.
```

Comment:
Something that I figured out reading the code as well: it would match: `[xyz]`, `[something-XYZ]`, `[XYZ-something]`